src/main.ts

```diff
@@ -7,18 +7,25 @@ import 'bootstrap/dist/css/bootstrap.css'
 import 'bootstrap-vue/dist/bootstrap-vue.css'
 import Resource from 'vue-resource'
 import TreeView from 'vue-json-tree-view'
-
 import './registerServiceWorker'
+import 'authenticationPlugin/firebaseInit'
```

Comment:
Nothign specific to firebase should be outside the plugin. Everythign should happen as a part of authenticationPlugin